frontend/lib/about/texts.dart

```diff
@@ -47,9 +49,21 @@ List<Widget> getDisclaimerText(BuildContext context) {
 }
 
 List<Widget> getPublisherText(BuildContext context) {
-  return toWidgets(Theme.of(context), [
-    Paragraph(
-      content: buildConfig.publisherText,
-    )
-  ]);
+  return [
+    Html(
+      data: buildConfig.publisherText,
+      style: {
+        'p': Style(margin: Margins.only(bottom: 10), fontSize: FontSize(15)),
+        'h4': Style(margin: Margins.only(bottom: 10), fontSize: FontSize(15)),
+        'li': Style(padding: HtmlPaddings.only(left: 10), fontSize: FontSize(15)),
+        'ul': Style(padding: HtmlPaddings.only(left: 20), fontSize: FontSize(15)),
+        'a': Style(padding: HtmlPaddings.only(left: 10), fontSize: FontSize(15))
```

Comment:
```suggestion
        '*': Style(fontSize: FontSize(15), letterSpacing: 0.1),
        'p': Style(margin: Margins.only(bottom: 10)),
        'h4': Style(margin: Margins.only(bottom: 10)),
        'li': Style(padding: HtmlPaddings.only(left: 10)),
        'ul': Style(padding: HtmlPaddings.only(left: 20)),
```
not sure if `'a': Style(padding: HtmlPaddings.only(left: 10)` is needed, I don't see any difference

Reply:
Ah that's a good improvement. I Was looking for that general styling (*) approach

Reply:
@seluianova i see a slight improvement with the letter spacing, even its not perfect.
But i think for an imprint its okay now how it is. In my opinion you can merge this

Reply:
Thanks a lot for your cooperation :)